edx_proctoring/static/proctoring/js/views/proctored_exam_attempt_view.js

```diff
@@ -174,6 +174,7 @@ var edx = edx || {};
             if (!confirm(gettext('Are you sure you want to remove this student\'s exam attempt?'))) {
                 return;
             }
+            $(this.el).css('cursor', 'wait');
```

Comment:
One thing to test is if this style setting will be global to the whole page... try moving the cursor around and see if it changes back to pointer.

If it does change, I think you want to set this style on the whole HTML body. See edx_proctoring/templates/proctored_exam/entrance.html